backend/models/sprint.js

```diff
@@ -0,0 +1,42 @@
+module.exports = (sequelize, Sequelize) => {
+  const Sprint = sequelize.define('sprint', {
+    id: {
+      type: Sequelize.INTEGER,
+      primaryKey: true,
+      autoIncrement: true
+    },
+    start_date: {
+      type: Sequelize.DATE,
+      allowNull: true
+    },
+    end_date: {
+      type: Sequelize.DATE,
+      allowNull: true
+    },
+    sprint: {
+      type: Sequelize.INTEGER,
+      allowNull: false
+    },
+    created_at: {
+      type:Sequelize.DATE,
+      allowNull: false
+    },
+    updated_at: {
+      type: Sequelize.DATE,
+      allowNull: false
+    }
+  },
+  {
+    underscored: true
+  })
+
+
+  Sprint.associate = (models) => {
+    Sprint.belongsTo(models.Group, {
+      foreignKey: 'group_id',
+      as: 'group',
+    })
+  }
+
+  return Sprint
+}
```

Comment:
Add newline at end of file

https://gist.github.com/camh-/1bebfcff1b0f814e9b191edc60d5206b